actions.js

```diff
@@ -1,4 +1,6 @@
 let IMAGES = []; // Will put all images in this
+let pickedImageHistory = []; // Will put all picked images in this
+let roundsPlayed = 0; // Amount of rounds played
```

Comment:
I would propose to remove this variable, since it's equal to pickedImageHistory's length, right?